content/glossary/gitbom_id_embedding.md

```diff
@@ -0,0 +1,4 @@
++++
+title = "GitBOM ID Embedding"
+toc=true
++++
```

Comment:
Empty file?

Reply:
Yes... needs to be filled in (I should have left a TODO comment).  Mostly I wanted to get stuff out where others could comment.